inst/extdata/install-packages.R

```diff
@@ -113,45 +92,135 @@ installEsqLabsR <- function() {
                  build = TRUE
   )
 
-  # Only for local installation
-  if (installOption == 1) {
+  cli_progress_done(result = "done")
+}
 
-    cli_progress_step("Getting minimal version of PKSim.")
-    getPKSimMinimal()
+load_esqlabsR <- function(){
+  cli_progress_step("Load esqlabsR")
+  unloadNamespace(ns = "esqlabsR")
+  unloadNamespace(ns = "ospsuite.parameteridentification")
+  # test if installed packages can be loaded
+  library("esqlabsR")
 
-    cli_progress_step("Verifying PKSim installation")
-    # test if PK-Sim.R dll can be loaded
-    initPKSim()
+  cli_progress_done(result = "done")
+}
 
-    snapshot(prompt = FALSE)
-  }
+run_test_simulation <- function() {
+  # Run test simulation to verify installation
 
-  cli_progress_step("Loading esqlabsR")
-  # test if installed packages can be loaded
-  library("esqlabsR")
+  cli_progress_step("Run test simulation.")
 
+  tryCatch(expr = {
+    require(esqlabsR)
 
-  cli_progress_step("Running test simulation to verify installation.")
-  # run test simulations if the working directory path ends with "Code"
-  if (endsWith(getwd(), "Code")) {
-    runTestSimulation()
-    cli_progress_done()
-  } else {
-    cli_alert_warning("Test simulation files could not be found.")
+    suppressWarnings({
+      projectConfiguration <- createDefaultProjectConfiguration(path = esqlabsR:::test_ProjectConfiguration())
+    })
+    # Define which scenarios to run
+    scenarioNames <- c("TestScenario")
+    # Set scenario names to NULL if you want to simulate all scenarios defined in the
+    # excel file
+    # scenarioNames <- NULL
+
+    # Create `ScenarioConfiguration` objects from excel files
+    scenarioConfigurations <- readScenarioConfigurationFromExcel(
+      scenarioNames = scenarioNames,
+      projectConfiguration = projectConfiguration
+    )
+
+    scenarios <- createScenarios(scenarioConfigurations)
+
+    simulatedScenarios <- runScenarios(scenarios = scenarios)
+
+    cli_progress_done(result = "done")
+    cli_alert_success("Installation successful.")
+
+  },
+  error = function(error){
     cli_progress_done(result = "failed")
+    cli_alert_danger(text = paste("Simulation test failed: ", error))
+  })
+}
+
+initialize_project <- function(){
+
+  initialize_option <- utils::menu(c("Yes", "No"),
+                                   title = "Do you want to initialize project folder structure ?"
+  )
+
+  if (initialize_option == 1) {
+    cli_progress_step("Initialize project structure")
+    init_project(destination = "../")
+    update_project_conf_file()
+    cli_progress_done(result = "done")
   }
 
-  cli_alert_success("Installation successful.")
+}
+
+update_project_conf_file <- function(){
```

Comment:
@Felixmil The file `TimeValuesData` must also be renamed to ´myProject_TimeValuesData.xlsx`
![image](https://github.com/esqLABS/esqlabsR/assets/25903040/13846802-1120-492b-a7fc-a5ea67f92d0d)